script.py

```diff
@@ -0,0 +1,52 @@
+import os
+import re
+import pysrt
+import logging
+from pathlib import Path
+from pytube import YouTube
+
+logger = logging.getLogger(__name__)
+
+
+class Captions:
+    def __init__(self, URL):
+        self.yt = YouTube(URL)
+        self.video_path = os.getcwd() + "/videos/{}".format(
+            self.yt.title.lower().replace(" ", "_")
+        )
+        Path(self.video_path).mkdir(parents=True, exist_ok=True)
+        self.srt_filename = self.create_srt()
+        self.generate_script()
+
+    def create_srt(self):
+        try:
+            caption_dict = self.yt.captions.lang_code_index
+            en_key = None
+            for key in caption_dict.keys():
+                if re.findall("^en.*", key):
+                    en_key = key
+                    break
+
+            if en_key:
+                self.yt.captions[en_key].download("captions")
+            elif caption_dict.get("a.en", None):
```

Comment:
Maybe this:

```suggestion
            elif "a.en" in caption_dict:
```

Reply:
Would not work, we need None to be returned in case there isn't a "a.en" key